server/router.es6

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+const express = require('express');
+
+const commonController = require('./controllers/common.es6');
+const middleware = require('./middleware.es6');
+const noncommercialController = require('./controllers/noncommercial.es6');
+const passportConfig = require('./auth/passport-config.es6');
+const tempOutfitterController = require('./controllers/temp-outfitter.es6');
+
+const router = express.Router();
+
+/* allow any server to check the preflights */
+router.options('*', middleware.setCorsHeaders, (req, res) => {
+  res.set('Access-Control-Allow-Headers', 'accept, content-type');
+  res.set('Access-Control-Allow-Methods', 'GET, PUT, POST, DELETE, OPTIONS, PATCH');
```

Comment:
I'm not sure I know what the `OPTIONS` and `PATCH` methods do. Do we need them for any of our routes?